lib/execution_engine2/sdk/EE2Runjob.py

```diff
@@ -192,15 +193,20 @@ def _check_workspace_permissions(self, wsid):
                 )
 
     def _check_workspace_permissions_list(self, wsids):
-        perms = self.sdkmr.get_workspace_auth().can_write_list(wsids)
-        bad_ws = [key for key in perms.keys() if perms[key] is False]
-        if bad_ws:
-            self.logger.debug(
-                f"User {self.sdkmr.user_id} doesn't have permission to run jobs in workspace {bad_ws}."
-            )
-            raise PermissionError(
-                f"User {self.sdkmr.user_id} doesn't have permission to run jobs in workspace {bad_ws}."
+        # TODO Cover this in tests once you can execute multiple independent runs
```

Comment:
Yes, when retrying multiple jobs they can all have different workspace ids.

Reply:
Hmm... when would that actually happen though? A user is typically working in a single workspace / narrative at a time.

Reply:
When calling the api directly you can pass in any job ids you want for retry_multiple(). It seems wrong to not account for this behavior without this or preventing passing in multiple wsids to retry_multiple()

Reply:
I'm confused then - if you can submit multiple workspace IDs via `retry_multiple` why can't this function be tested?

Reply:
Because right now they are calling run_job one at a time, and that one causes a seperate ws lookup for each job, instead of in 1 ws call, like this section will expect.

Reply:
`they` = `retry_multiple` I assume. Ok, that makes sense